v1/protocols/hyper.ts

```diff
@@ -64,11 +64,13 @@ export class HyperProtocol implements Protocol<Static<typeof HyperProtocolFields
 
     ctx?.logger.info(`[hyper] Published: ${link}`)
 
+    const dnslink = raw.substring('hyper://'.length)
```

Comment:
```suggestion
    const dnslink = `/hyper/${raw.substring('hyper://'.length)}/`
```

Unless I'm mistaken, we need the `/hyper/` prefix added here.

Reply:
ah good catch! not sure how I didn't get that earlier